base_tier_validation_forward/models/tier_review.py

```diff
@@ -1,53 +1,31 @@
 # Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class TierReview(models.Model):
     _inherit = "tier.review"
     _order = "sequence"
 
-    name = fields.Char(compute="_compute_definition_data", store=True)
+    name = fields.Char(related="definition_id.name")
```

Comment:
Standard use is `related` already.
but we changed it to a computed field because we need to update the name in `tier.review`

if we use a related field, the name won't change after forwarding.

Steps to test:
1. Click Forward > add next reviewer
2. Add a Forward Description > Forward
![Selection_008](https://github.com/user-attachments/assets/a89c6a91-095e-4cae-ba7c-c6f1b470e656)
3. The description is blank because it is only related to definition_id
![Selection_009](https://github.com/user-attachments/assets/c867f441-5fac-4ffe-a3cc-2b1cf1681892)